django-verdant/rca/templates/rca/base.html

```diff
@@ -1,11 +1,11 @@
 {% load compress static wagtailimages_tags wagtailuserbar rca_tags wagtailsettings_tags wagtailcore_tags %}
 {% get_settings %}
 <!DOCTYPE html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="no-js lt-ie10 lt-ie9"> <![endif]-->
-<!--[if IE 9]>         <html class="no-js lt-ie10"> <![endif]-->
-<!--[if gt IE 9]><!--> <html class="no-js"> <!--<![endif]-->
+<!--[if lt IE 7]>      <html class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7" lang="en-US"> <![endif]-->
```

Comment:
Might `en-GB` be preferable, or even `en`?

Reply:
I've used US to be consistent with the new pages - I wanted to ask Nick why he'd chosen that but he's quite busy today... so have just made this the same for consistency across the site for now.